src/wrapper/signature.rs

```diff
@@ -44,13 +45,14 @@ pub enum JavaType {
     Method(Box<TypeSignature>),
 }
 
-impl JavaType {
-    /// Parse a type string into a JavaType enum.
-    pub fn from_str(s: &str) -> Result<JavaType> {
+impl FromStr for JavaType {
```

Comment:
Just a question: is it a breaking change — the fact that we implement a method from a trait/not declare it in this type?

Reply:
To use the method `from_str`, there should be added a `use std::str::FromStr;` statement, in a module where `from_str` is called.

Reply:
Therefore it is breaking if `JavaType#from_str` is public, isn't it?

Reply:
Yes, exactly.

Reply:
I'm going to update the changelog. :sweat_smile: